src/app/api/image-proxy/route.ts

```
@@ -0,0 +1,70 @@
+import { NextRequest, NextResponse } from "next/server"
+import sharp from "sharp"
+
+export const dynamic = "force-dynamic"
+
+export async function GET(request: NextRequest): Promise<NextResponse> {
+    try {
+        // URL 파라미터에서 이미지 URL 추출
+        const { searchParams } = new URL(request.url)
+        const imageUrl = searchParams.get("url")
+        const width = parseInt(searchParams.get("width") || "800", 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)